service/raft/group0_state_machine.hh

```diff
@@ -46,14 +46,22 @@ struct topology_change {
     std::vector<canonical_mutation> mutations;
 };
 
+struct mixed_change {
```

Comment:
Needs explanatory comment.

Reply:
Added, plz let me know if it's enough, I just referenced to the 2 other structs. If that's not explanatory enough, maybe the 2 other structs should have some comment, because this newly introduced struct merely aggregates both.

Reply:
> Needs explanatory comment.

Needs a better name as well.